Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs

```diff
@@ -273,6 +276,17 @@ public static float[] IntoMat3x4f(Matrix4x4 mat)
             return IntoMat3x4f(new[]{mat});
         }
 
+        public static float[] IntoVec2f(Vector2[] vecs)
```

Comment:
Are U sure name of this method is proper?

Reply:
Yes. It converts Vector2[] into Vec2f[] (RGL type that consists of two floats).